src/components/GrowerDetail.js

```diff
@@ -241,10 +223,11 @@ const GrowerDetail = ({ open, growerId, onClose }) => {
           awaitingCount,
           rejectedCount,
         ] = await Promise.all([
-          getCaptureCountGrower(true, true, grower.id),
-          getCaptureCountGrower(true, false, grower.id),
-          getCaptureCountGrower(false, false, grower.id),
+          getCaptureCountGrower('approved', grower.id),
```

Comment:
Can we create some `const`s to avoid the need to put `status` string literals in the code?